src/main/java/net/helix/pendulum/controllers/RoundViewModel.java

```diff
@@ -438,7 +438,7 @@ public static void updateApprovees(Tangle tangle, TransactionValidator transacti
             approve.addHash(lastTx.getHash());
             approve.store(tangle);
         }
-        transactionValidator.updateStatus(TransactionViewModel.fromHash(tangle, lastTx.getHash()));
+        //transactionValidator.updateSolidityStatus(TransactionViewModel.fromHash(tangle, lastTx.getHash()));
```

Comment:
so update status is totally removed from updateApprovees?

Reply:
Updated the PR. However, the current approach for milestone solidification is still suboptimal - subject for further work